src/codegen/generator.py

```diff
@@ -45,3 +45,7 @@ def generate(dest):
         _generate_file(
             codegen_metadata, info["templateFile"], dest / info["relativeOutputPath"]
         )
+    
```

Comment:
Is this how to run the code generator? `poetry run python -m codegen --dest .\generated\nidaqmx`

We should add that to CONTRIBUTING.md.

Reply:
This is more relevant for https://github.com/ni/nidaqmx-python/pull/214